frontend/resourceadm/pages/AboutResourcePage/AboutResourcePage.tsx

```diff
@@ -85,24 +80,11 @@ export const AboutResourcePage = ({
   /**
    * Resource type options
    */
-  const resourceTypeOptions = Object.keys(resourceTypeMap).map(key => ({
+  const resourceTypeOptions = Object.keys(resourceTypeMap).map((key) => ({
     value: key,
-    label: resourceTypeMap[key]
+    label: resourceTypeMap[key],
```

Comment:
Since you depend on the key and the value, you can use `Object.entries. `Then you do not need to do the hash-map-lookup at line 85. 😊 

```
const resourceTypeOptions = Object.entries(resourceTypeMap).map(([key, value]) => ({
    value: key,
    label: value,
  }));
```

Reply:
Smart!